internal/service/users/user.go

```diff
@@ -97,3 +97,21 @@ func (s *Service) findUser(user *types.User) (*types.User, error) {
 	}
 	return user, types.NewErrServerError(result.Error)
 }
+
+func (s *Service) CreateUserSponsorship(userId uuid.UUID, sponsorId uuid.UUID) (types.UserSponsorship, error) {
+	userSponsorship := types.UserSponsorship{
+		UserID:    userId,
+		SponsorID: sponsorId,
+	}
```

Comment:
My gut feeling was that this would be in `invite.go`. But I don't feel strongly. Any thoughts @t-young31 ?